tests/test_icml_conference.py

```diff
@@ -1433,3 +1452,124 @@ def test_decision_stage(self, openreview_client, helpers):
         assert openreview_client.get_invitation('ICML.cc/2023/Conference/Submission3/-/Decision')
         assert openreview_client.get_invitation('ICML.cc/2023/Conference/Submission4/-/Decision')
         assert openreview_client.get_invitation('ICML.cc/2023/Conference/Submission5/-/Decision')
+
+    def test_forum_chat(self, openreview_client, helpers):
+
+        openreview_client.post_invitation_edit(
+            invitations='ICML.cc/2023/Conference/-/Edit',
+            readers = ['ICML.cc/2023/Conference'],
+            writers = ['ICML.cc/2023/Conference'],
+            signatures = ['ICML.cc/2023/Conference'],
+            invitation = openreview.api.Invitation(
+                id = 'ICML.cc/2023/Conference/-/Submission',
+                reply_forum_views = [
+                    {
+                        'id': 'all',
+                        'label': 'All'
+                    },
+                    {
+                        'id': 'discussion',
+                        'label': 'Discussion',
+                        'filter': '-invitations:ICML.cc/2023/Conference/Submission${note.number}/-/Chat',
```

Comment:
what does the "-" mean here?

Reply:
This means that the view should hide notes that are replies of the listed invitations.